frontend/packages/ux-editor/src/components/config/componentSpecificContent/Summary2/Override/Summary2OverrideDisplaytype.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { useEffect } from 'react';
+import { StudioCard, StudioNativeSelect } from '@studio/components';
+import type {
+  Summary2OverrideConfig,
+  SummaryCustomTargetType,
+} from 'app-shared/types/ComponentSpecificConfig';
+import { useCustomConfigType } from './hook/useCustomConfigType';
+import { useTranslation } from 'react-i18next';
+
+export interface Summary2OverrideDisplaytypeProps {
+  override: Summary2OverrideConfig;
+  onChange: (override: Summary2OverrideConfig) => void;
+}
+
+export const Summary2OverrideDisplaytype = ({
+  override,
+  onChange,
+}: Summary2OverrideDisplaytypeProps) => {
+  const customConfigType = useCustomConfigType();
```

Comment:
Maybe we can change it to the following to increase readability? 😄 
```suggestion
  const customConfigTypes: CustomConfigType[] = useCustomConfigType();
```

Reply:
Yes based on your suggistion for creating CustomConfigTypeOptions then we need to import the type here🙂, I fixed